Jenkinsfile

```diff
@@ -1,5 +1,7 @@
 @Library('shared-libraries') _
 
+def Email = "rob.rudin@marklogic.com"
```

Comment:
Is there a way to do this via an environment variable? I'd rather not have my work email address sitting around like this.

Reply:
Sure, I'll pass it as a parameter.